server/core/src/main/java/org/infinispan/server/core/dataconversion/JBossMarshallingTranscoder.java

```diff
@@ -85,6 +86,8 @@ public Object transcode(Object content, MediaType contentType, MediaType destina
       }
       if (destinationType.equals(APPLICATION_UNKNOWN)) {
          try {
+            //TODO: Remove wrapping of byte[] into WrappedByteArray from the Hot Rod Multimap operations.
```

Comment:
Is there a jira for this todo?

Reply:
Not yet, I'll create it

Reply:
https://issues.jboss.org/browse/ISPN-9458